### DISCUSSION TITLE
Fans start and stop on their own

### DISCUSSION THREAD

**Discussion by dude542** [devcompl/TTController#109] [Q&A] [answered]
Hello!
For whatever reason, when I start the service, it will work for a few seconds, and then it will seemingly stop working and go to the default Thermaltake Rainbow Color. Then, as time progresses it will go back to the color that was programmed in the config file again. It will then continue to go back and forth. Any idea what could be causing this? Also, for some reason, when the service is working, the lights are dim even though the full RGB value is used (i.e 255,0,0 appears as dim red).

**Comment by MoshiMoshi0** [comment, marked as answer]
> For whatever reason, when I start the service, it will work for a few seconds, and then it will seemingly stop working and go to the default Thermaltake Rainbow Color.

https://moshimoshi0.github.io/TTController/config/port#ignorecolorcache

> Also, for some reason, when the service is working, the lights are dim even though the full RGB value is used (i.e 255,0,0 appears as dim red).

No idea. What fans and effect do you use. Is full blue/green also dim?

**Reply by dude542** [reply]
So after troubleshooting for a while longer, I just started over with a config that you gave another user on a separate issue and it seemed to work perfectly. That config also had the IgnoreColorCache in it too so there was no flickering going on anymore. I'm going to chalk the dimness up to there being a separate error in the config file that I was using previously as the one I am using now works perfectly.
